packages/react/src/components/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Alert } from './Alert';
+
+describe('Alert', () => {
+  test('has correct icon title when severity="danger"', (): void => {
+    render(<Alert severity='danger'>Alert me!</Alert>);
+    screen.debug();
```

Comment:
Should we remove the debug before merging? :) Personally, I think the debug is OK for developers to add locally when needed.

Reply:
Ah! That was not supposed to be committed 😅 I was looking for ways to test if correct icon was being picked, but couldn't think of any better way than using `title`.

Will remove it!

Reply:
I think that using the getByTitle is correct as long the title is set on the SVG. Alternatively, we could use the aria-label and getByLabelText method.


_"Text in a <title> element is not rendered as part of the graphic, but browsers usually display it as a tooltip. If an element can be described by visible text, it is recommended to reference that text with an [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) attribute rather than using the <title> element."_

Source: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title